.travis.yml

```diff
@@ -18,6 +18,8 @@ script:
 - if [ ${REPO_TYPE} == "r" ]; then cd tests && travis_wait pytest --cov repo2docker
   -v ${REPO_TYPE}; else cd tests && travis_retry pytest --cov repo2docker -v ${REPO_TYPE};
   fi
+- pip install -r docs/doc-requirements.txt
```

Comment:
I'm not sure that these two lines are using the correct directory or if the script is still in the `tests` directory when running. Going to play a bit with my fork and this PR.

Reply:
Good point. I was assuming that `cd tests && pytest` would leave me back in the original directory.